x-pack/legacy/plugins/reporting/server/usage/get_reporting_usage.ts

```diff
@@ -38,25 +39,39 @@ const PRINTABLE_PDF_JOBTYPE = 'printable_pdf';
 const getKeyCount = (buckets: KeyCountBucket[]): { [key: string]: number } =>
   buckets.reduce((accum, { key, doc_count: count }) => ({ ...accum, [key]: count }), {});
 
-function getAggStats(aggs: AggregationResults) {
-  const { buckets: jobBuckets } = aggs[JOB_TYPES_KEY] as AggregationBuckets;
-  const jobTypes: JobTypes = jobBuckets.reduce(
+// indexes some key/count buckets by statusType > appName > jobType: statusCount
```

Comment:
Maybe I'm misreading the function but it seems like it's statusType > jobType > appName, right?

Reply:
From the snapshot:

```
 "statuses": Object {
      "completed_with_warnings": Object {
        "PNG": Object {
          "dashboard": 1,
        },
      },
    },
```

Reply:
You are absolutely right - this comment had been written using an early data model I was working with :)